jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -76,13 +76,24 @@ def test_model_ids(cubeviz_helper, spectral_cube_wcs):
         plugin.vue_add_model({})
 
 
+@pytest.mark.skip(reason="Needs #3156 after merging #3190")
```

Comment:
I don't quite understand "Needs #3156 after merging #3190". #3156 is this PR. So does this mean we can unskip this here after #3190 is merged?